### ISSUE TITLE
section anchors

### ISSUE THREAD

**Issue by bambooforest** [phoible/phoible.github.io#7]
unless i missed it, it would be great to be able to refer to specific sections via URL in our conventions, e.g.

http://phoible.github.io/conventions#dipthongs

**Comment by drammock** [comment]
agree, would be good.  It's supposed to be possible but so far I haven't
figured out the magic incantation to make it actually work.

**Comment by bambooforest** [comment]
This seems to work when we mention a section in the text, e.g.:

http://phoible.github.io/conventions/#frictionalized-sounds

So perhaps adding a TOC would do it.

**Comment by drammock** [comment]
Ah, I was stuck on why https://phoible.github.io/conventions/#glottalization,-laryngealization,-and-creaky-voicing didn't work.  But I just tried deleting the commas and it works.  So we just have to remove the commas there.
